### PR TITLE
Updated CO2Calc submodule to dev branch of CO2 Calc module

### PR DESCRIPTION
**Pull request by sami1riaz**
Updated all imports from Co2Calc dev branch.

.gitmodules is pointing to dev branch of co2calc on this branch

Removed use of Unit in Heating class (models.py) as it no longer exists in co2calc. Not sure if we need to completely remove this (from the frontend as well) or replace it with something

Note: All the doc files can be ignored for this review as I have simply replaced the class names to the new ones in there

### CODE REVIEW

File: frontend/lib/ga/index.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
 // log the pageview with their URL
 export const pageview = (url) => {
-  window.gtag('config', process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS, {
-    page_path: url,
-  })
+  if (typeof window.gtag !== 'undefined') {
```
Comment (sami1riaz) [inline frontend/lib/ga/index.js:3]: I was getting a "window.gtag is not a function" error when I submitted the demo mode dataforms, so i added this check after reading this: https://stackoverflow.com/questions/62791299/typeerror-window-gtag-is-not-a-function

File: backend/src/emissions/models.py [hunk 2]
```diff
@@ -320,8 +318,8 @@ class Heating(models.Model):
     )
     fuel_type_choices = [(x.name, x.value) for x in HeatingFuel]
     fuel_type = models.CharField(max_length=20, choices=fuel_type_choices, blank=False)
-    unit_choices = [(x.name, x.value) for x in Unit]
-    unit = models.CharField(max_length=20, choices=unit_choices, blank=False)
+    # unit_choices = [(x.name, x.value) for x in Unit]
```
Comment (redfrexx) [inline backend/src/emissions/models.py:321]: I think we still need the unit attribute, but need to define it manually for now. Can you add the following code?

```
unit_choices = [(x, x) for x in ["kWh", "l", "kg", "m^3"]]
unit = models.CharField(max_length=20, choices=unit_choices, blank=False)
´´´

File: frontend/lib/ga/index.js (hunk 1 above)
Comment (redfrexx) [reply frontend/lib/ga/index.js:3]: sounds good

**Review by redfrexx — CHANGES_REQUESTED** [review]
Looks good! Just one small adaption necessary and it should be good to be merge.

File: frontend/lib/ga/index.js (hunk 1 above)
Comment (codingfabi) [reply frontend/lib/ga/index.js:3]: I think we could also avoid this problem by removing google analytics from our packages in during https://github.com/pledge4future/WePledge/issues/131

File: backend/src/emissions/models.py (hunk 2 above)
Comment (redfrexx) [reply backend/src/emissions/models.py]: fixed

**Review by redfrexx — APPROVED** [review]

**Pull request merged**